package.json

```diff
@@ -288,9 +288,11 @@
     "vscode": "1.1.10"
   },
   "dependencies": {
+    "@types/lodash": "^4.14.102",
     "cobertura-parse": "1.0.4",
     "glob": "7.1.2",
     "lcov-parse": "1.0.0",
+    "lodash": "^4.17.5",
```

Comment:
Can you peg these at a concrete version like the other dependencies, it will make it easier for me to get consistent builds when I make the next release.

Reply:
Sure!